api/src/integrations/epic/datasource.js

```diff
@@ -58,7 +64,10 @@ class DataSource {
       }
 
       await Promise.all(promises);
+      this.status = { ...this.status, ...this.getStatusTotals() };
```

Comment:
I think something got messed up during rebase for this file.

The current version in master is different.  It does batch requests to EPIC (to avoid blowing them up).
And for example, getStatusTotals() no longer exists in master, and doesn't exist here either (not sure why its not throwing an error).